crates/cdk/src/publish.rs

```diff
@@ -215,15 +222,25 @@ fn from_connectorvis(cv: &ConnectorVisibility) -> PkgVisibility {
 }
 
 pub(crate) fn find_connector_toml(package_info: &PackageInfo) -> Result<PathBuf> {
-    let smartmodule_toml = package_info.package_relative_path(CONNECTOR_TOML);
+    let connector_toml = package_info.package_relative_path(CONNECTOR_TOML);
 
-    if smartmodule_toml.exists() {
-        return Ok(smartmodule_toml);
+    if connector_toml.exists() {
+        return Ok(connector_toml);
     }
 
     Err(anyhow::anyhow!("No \"{}\" file found", CONNECTOR_TOML))
 }
 
+pub(crate) fn find_readme_md(package_info: &PackageInfo) -> Result<PathBuf> {
+    let readme_md = package_info.package_relative_path(README_MD);
```

Comment:
This assumes the Readme is in the same file than Connector.toml? Can we add capability to override where the readme is?

Reply:
I think we could have a param for the publish command which takes the path and uses that if provided? Otherwise fallback to this?

Reply:
I think this connector: https://github.com/infinyon/http-sink-connector is a good use case for this, wdyt? 

Reply:
Sounds good to me. Also for instance, in the kafka connector we right now have a big readme for both source and sink. Maybe we should split them and put them on each connector directory

Reply:
Great insights! Will provide the option!